src/en/getting-started-jaas.md

```diff
@@ -0,0 +1,127 @@
+Title: Getting started with Juju as a Service
+
+# Getting started with Juju as a Service
+
+Juju as a Service (JAAS) is the fastest and easiest way to quickly get started
```

Comment:
fastest ANd quickly?

I would just say fastest and easiest way to model and deploy your cloud-based applications